### PR TITLE
use Jest as test framework #10

### PR DESCRIPTION
**Pull request by dohamokdad**
$ npm install --save-dev jest
$ npm run test

PASS  ./sum.test.js
✓ adds 1 + 2 to equal 3 (5ms)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,3 +9,8 @@
   },
   "homepage": "https://github.com/NourSammour/biletiniz#readme"
 }
+{
+  "scripts": {
+    "test": "jest"
```
Comment (NourSammour) [inline package.json:14]: you didn't add jest to dependencies

**Review by NourSammour — CHANGES_REQUESTED** [review]

**Review by dohamokdad — COMMENTED** [review]
Which dependencies?

File: package.json (hunk 1 above)
Comment (dohamokdad) [reply package.json:14]: Which dependencies?

**Comment by NourSammour** [comment]
i should see jest in project dependencies

**Pull request closed without merging**